mmv1/products/compute/api.yaml

```diff
@@ -8553,6 +8553,21 @@ objects:
         description: |
           The target service url used to set up private service connection to
           a Google API or a PSC Producer Service Attachment.
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'network'
+        resource: 'Network'
+        imports: 'selfLink'
+        description: |
+          This field is only used for PSC.
+          The URL of the network to which all network endpoints in the NEG belong. Uses
+          "default" project network if unspecified.
```

Comment:
Is the value "default" returned by the API? Or does it imply using the default network and not send any value back?

Reply:
Yes, if you don't specify a network in the call, [the API](https://cloud.google.com/compute/docs/reference/rest/v1/regionNetworkEndpointGroups/insert#request-body) will try to use the `default` network instead.